tests/test_requests.py

```diff
@@ -2264,6 +2264,25 @@ def test_json_encodes_as_bytes():
     assert isinstance(p.body, bytes)
 
 
+def test_params_encodes_quote():
+    # in python 3 params are encoded using quote in place of quote_plus (Spaces are encoded as %20 in place of '+')
+    params = {"key": "value1 value2"}
+    p = PreparedRequest()
+    p.prepare(
+        method='GET',
+        url='https://www.example.com/',
+        params=params
+    )
+    if is_py2:
```

Comment:
```python
expected = 'https://www.example.com/?key=value1%20value2'
if is_py2:
     expected = 'https://www.example.com/?key=value1+value2'
assert p.url == expected
```
Please check you code locally first.

Reply:
And could you use the same example as shown on the original issue?

Reply:
I have modified and used same text as shown in the issue.

Reply:
Today , due to some work related constraint, I am not able to check it locally.